packages/bruno-app/src/components/ResponsePane/Timeline/index.js

```diff
@@ -33,9 +31,9 @@ const Timeline = ({ request, response }) => {
           );
         })}
 
-        {requestData ? (
+        {request.data ? (
           <pre className="line request">
-            <span className="arrow">{'>'}</span> data {requestData}
+            <span className="arrow">{'>'}</span> data <pre>{request.data}</pre>
```

Comment:
This causes a horizontal overflow, if the text is too long

Reply:
Nice catch. Fixed